hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -3046,6 +3047,54 @@ public void testPersistence() {
         assertEquals(240, persistenceConfig.getRebalanceDelaySeconds());
     }
 
+    @Override
+    @Test
+    public void testDevice() {
+        String deviceName = "amazing_device";
+        String baseDir = "base-directory";
+        int blockSize = 2048;
+
+        String xml = HAZELCAST_START_TAG
+                + "<device>"
+                + "    <device-name>" + deviceName + "</device-name>"
+                + "    <base-dir>" + baseDir + "</base-dir>"
+                + "    <block-size>" + blockSize + "</block-size>"
+                + "</device>\n"
+                + HAZELCAST_END_TAG;
+
+        Config config = new InMemoryXmlConfig(xml);
+        DeviceConfig deviceConfig = config.getDeviceConfig();
+
+        assertEquals(deviceName, deviceConfig.getDeviceName());
+        assertEquals(new File(baseDir).getAbsolutePath(), deviceConfig.getBaseDir().getAbsolutePath());
+        assertEquals(blockSize, deviceConfig.getBlockSize());
+    }
+
+    @Override
+    @Test
+    public void testTieredStore() {
+        String xml = HAZELCAST_START_TAG
+                + "<map name=\"my-map\">"
+                + "    <tiered-store enabled=\"true\">"
+                + "        <memory-tier>"
+                + "            <capacity>1024 MB</capacity>"
```

Comment:
Where will `pageSize` be configured? Are we going to use the `pageSize` of the `native-memory`? Same question for read-only/mutable page ratio.

Reply:
(in-memory) pageSize, read-only/mutable region ratio configuration are 'too internal' and we will go with some defaults for them for now, afaik. @petrpleshachkov wdyt?

Reply:
(for any future reader): We discussed this on the standup, for now we decided to keep the above config internal